extensions/wikia/ContentFeeds/ContentFeeds.class.php

```diff
@@ -219,13 +219,13 @@ public static function dummyRecentImagesParserHook( $input, $args, $parser ) {
 		return "Życie..";
 	}
 
-	public static function specialNewImagesHook( $images, $gallery, $limit ) {
+	public static function specialNewImagesHook( $images ) {
 		global $wgRequest;
 
 		if( $wgRequest->getVal('feed') == 'rss' ) {
 			//filter images and get the file url
 			$filteredImages = array();
```

Comment:
Old school array!  (OSA).  Definitely on board with updating these :).  Were you updating all occurrences in the files you touched or only the specific lines of code you changed?